packages/pangea-node-sdk/src/services/store.ts

```diff
@@ -0,0 +1,337 @@
+      );
+    }
+
+    return this.request.requestPresignedURL("v1beta/put", request);
+  }
+
+  /**
+   * @summary Update a file
+   * @description Update a file.
+   * @operationId store_post_v1beta_update
+   * @param {Store.UpdateRequest} request
+   * @returns {Promise} - A promise representing an async call to the update endpoint.
+   * @example
+   * ```js
+   * const request = {
+   *   id: "pos_3djfmzg2db4c6donarecbyv5begtj2bm",
+   *   path: "/",
+   *   remove_metadata: {
+   *     created_by: "jim",
+   *     priority: "medium",
+   *   }
+   *   remove_tags: ["irs_2023", "personal"],
+   * };
+   *
+   * const response = await client.update(request);
+   * ```
+   */
+  update(request: Store.UpdateRequest): Promise<PangeaResponse<Store.UpdateResult>> {
+    return this.post("v1beta/update", request);
+  }
+
+  /**
+   * @summary Create share links
+   * @description Create a share link.
+   * @operationId store_post_v1beta_share_link_create
+   * @param {Store.ShareLinkCreateRequest} request
+   * @returns {Promise} - A promise representing an async call to the share link create endpoint.
+   * @example
+   * ```js
+   * const authenticator = {
+   *   auth_type: Store.AuthenticatorType.PASSWORD,
+   *   auth_context: "my_fav_Pa55word",
+   * };
+   * const link = {
+   *   targets: ["pos_3djfmzg2db4c6donarecbyv5begtj2bm"],
+   *   link_type: Store.LinkType.DOWNLOAD,
+   *   authenticators: [authenticator],
+   * };
+   * const request = { links: [link] };
+   * const response = await client.shareLinkCreate(request);
+   * ```
+   */
+  shareLinkCreate(
+    request: Store.ShareLinkCreateRequest
+  ): Promise<PangeaResponse<Store.ShareLinkCreateResult>> {
+    return this.post("v1beta/share/link/create", request);
+  }
+
+  /**
+   * @summary Get share link
+   * @description Get a share link.
+   * @operationId store_post_v1beta_share_link_get
+   * @param {Store.ShareLinkGetRequest} request
+   * @returns {Promise} - A promise representing an async call to the share link get endpoint.
+   * @example
+   * ```js
+   * const request = { id: "psl_3djfmzg2db4c6donarecbyv5begtj2bm" };
+   * const response = await client.shareLinkGet(request);
+   * ```
+   */
+  shareLinkGet(
+    request: Store.ShareLinkGetRequest
+  ): Promise<PangeaResponse<Store.ShareLinkGetResult>> {
+    return this.post("v1beta/share/link/get", request);
+  }
+
+  /**
+   * @summary List share links
+   * @description Look up share links by filter options.
+   * @operationId store_post_v1beta_share_link_list
+   * @param {Store.ShareLinkListRequest} request
+   * @returns {Promise} - A promise representing an async call to the share link list endpoint.
+   * @example
+   * ```js
+   * const request = {};
+   * const response = await client.shareLinkList(request);
+   * ```
+   */
+  shareLinkList(
+    request: Store.ShareLinkListRequest = {}
```

Comment:
This one's example does the empty object thing again. Same feedback as above.

_But_ this was does exactly what I suggested above with regards to making the parameter optional with a default.

Reply:
Yes, it should be optional like in here.